pkg/utils/event_sender.go

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
I would make the event sender a package of `kubeless`? that way we can reuse it for all the triggers